src/main/java/com/gregtechceu/gtceu/api/machine/steam/SteamBoilerMachine.java

```diff
@@ -196,8 +196,7 @@ protected void updateCurrentTemperature() {
 
         if (getOffsetTimer() % 10 == 0) {
             if (currentTemperature >= 100) {
-                long fillAmount = (long) (getBaseSteamOutput() * (currentTemperature / (getMaxTemperature() * 1.0)) /
-                        2);
+                long fillAmount = (long) (getBaseSteamOutput() * ((float) currentTemperature / getMaxTemperature()));
```

Comment:
```suggestion
                long fillAmount = (long) (getBaseSteamOutput() * ((float) currentTemperature / getMaxTemperature()) / 2);
```
Small boilers output twice per second, so each fill creates half the base amount